### PR TITLE
Update the @actions/cache package to the latest version

### PR DESCRIPTION
**Pull request by smorimoto**
As the title says.

### CODE REVIEW

**Comment by smorimoto** [comment]
The main purpose of this PR is to update the @actions/cache for https://github.com/actions/toolkit/pull/533 and https://github.com/actions/cache/issues/326. But I think we can update other dependencies with this PR.

**Comment by smorimoto** [comment]
@aiqiaoy @joshmgross Could you review this PR?

File: package.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
...
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.2.0",
-    "@actions/exec": "^1.0.1",
-    "@actions/io": "^1.0.1",
-    "@actions/cache": "^1.0.1"
+    "@actions/core": "^1.2.4",
+    "@actions/exec": "^1.0.4",
+    "@actions/io": "^1.0.2",
+    "@actions/cache": "^1.0.2"
```
Comment (konradpabjan) [inline package.json:29]: Can we only update `@actions/cache`? We intentionally do not update all NPM packages to the latest versions unless required.

`npm update @actions/cache` and then `npm install` and `npm run release` should do the trick

Comment (smorimoto) [reply package.json:29]: I changed it.

**Pull request merged**